### DISCUSSION TITLE
Foxglove data disappear from plot panel

### DISCUSSION THREAD

**Discussion by Andre-ex9** [foxglove/community#825] [Q&A]
Hello,

I'm using Foxglove for visualize ROS2 live data. It worked fined for almost 4 weeks but now i have a problem that i can't solve.
I have several plot panels and the data displayed on it disappear : Every 5 seconds, the oldest points are deleted, so it can only display the last 15 seconds of the ploted data. 
I had no problems before as i displayed more than an hour of streamed data multiple times, but now something has changed and don't know why.

Has anyone faced the same issue ?
Tell me if you need more info on the situation.

**Comment by Andre-ex9** [comment]
![Capture d’écran du 2023-09-13 18-35-08](https://github.com/foxglove/community/assets/124618526/c97e7f62-179c-456a-9f8b-9ce17228837c)

Picture of the foxglove page for reference

**Comment by cfoust** [comment]
Hello! We made a mistake and started to cull points older than 10 seconds as a way of reducing memory usage. Though we still cull datasets larger than 15,000 messages, the time-based culling should be gone in the next release.

**Reply by Andre-ex9** [reply]
Thank you for your quick answer.

Do you know when the next release should be up ?

**Reply by cfoust** [reply]
Should be today or tomorrow!

**Reply by Doooittttt** [reply]
Hello! My large message seems to be culled. I have a large message that seems to have been culled due to being too large. Is there any way to prevent this message from being culled?

**Reply by defunctzombie** [reply]
@Doooittttt Please file a new bug report with steps to reproduce and some sample data here: https://github.com/foxglove/studio/issues/new/choose

**Comment by defunctzombie** [comment]
I'm gonna close this discussion as resolved. If you are still experiencing issues please file a bug report here: https://github.com/foxglove/studio/issues/new/choose and include the steps and sample data to reproduce.
